packages/runtime/src/runtimes/react/index.tsx

```diff
@@ -52,6 +54,8 @@ export interface ReactRuntime {
     elementTree: ReactPage.ElementData,
     replacementContext: ReactPage.SerializableReplacementContext,
   ): ReactPage.Element
+  registerBreakpoints(breakpoints: Breakpoints): void
```

Comment:
If we put this in the API handler, how will we retrieve breakpoints during SSR? Do we need the breakpoints for SSR rendering? We do to generate the classes with responsive styles, right? For the `getFonts` case it's just for the builder to request and show in the panel. But for the runtime, it's a bit more complicated than that.

Should breakpoints perhaps be a prop on the Makeswift page component? Or a piece of context?